release.md

```diff
@@ -0,0 +1,3 @@
+Release type: patch
+
+Internal GrapQLRouter FastAPI dependency is now async
```

Comment:
```suggestion
Release type: patch

This release changes an internal implementation for FastAPI's
GraphQL router. This should reduce overhead when using the context,
and it shouldn't affect your code.
```

can you rename this file to RELEASE.md? our check is case sensitive :D

Reply:
done.